day01/ex00/ex00.md

```diff
@@ -0,0 +1,114 @@
+# Exercise 00 - Linear Regression
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex00              |
+|   Files to turn in :    |  linear\_model.py  |
+|   Authorize module :    |  numpy, matplotlib |
+|   Forbidden module :    |  sklearn           |
+|   Forbidden function :  |  LinearRegression  |
+|   Remarks :             |  Read the doc      |
+
+**Objectives:** 
+
+* Reinforce the mathematical skills tackled in **Mathematical Delights**, especially the __matrix-matrix operations__ and __linear gradient__.
+* Be able to explain what is the __linear cost function__ and the __gradient descent__.
+
+* Be able to implement basic methods in order to perform a linear regressions.
+
+
+**Instructions:**
+
+You can find in the ressources a tiny data set called __are_blue_pills_magics.csv__ which give you mathematical performance of patients in function of the quantity of the "blue pills" they took before the test.
+
+As hypothesis function h, you will choose:
+$$
+h(X)= \theta_0x_0 + \theta_1x_1
+$$
+where $\pmb{X}$ is the vector $(x_0, x_1)$ and $\pmb{theta} = (theta_0, \theta_1)$
+
+
+You will model the data and plot 2 differents graphs:
+* The graph with the data and the best hypothesis you find for the spacecraft piloting score versus the quantity of "blue pills" (see example figure 1),
```

Comment:
May be we should be more explicit about the fact they have to fit it "by hand" ?